config/default.yml

```diff
@@ -4,3 +4,8 @@ Packaging/GemspecGit:
   Description: 'Use pure Ruby alternative instead of `git ls-files`.'
   Enabled: true
   VersionAdded: '0.86'
+
+Packaging/RequireRelativeToLib:
+  Description: 'Avoid using `require_relative` with relative path to lib.'
+  Enabled: true
+  VersionAdded: '0.89'
```

Comment:
Shouldn't this include `rubocop-packaging` versioning, instead of `rubocop`'s?

Reply:
Nope. It's a version+1 or something like that. I don't know the exact rule, really.
However, steps to reproduce that this `VersionAdded` is correct:
- `rubocop-extension-generator rubocop-packaging`
- `bundle exec rake 'new_cop[Packaging/RelativeRequireToLib]'`

`config/default.yml` should have the following content:
```
  # Write it!

  Packaging/RelativeRequireToLib:
    Description: 'TODO: Write a description of the cop.'
    Enabled: pending
    VersionAdded: '0.89'
```

Reply:
That sounds like a problem with the generator to me. You should include `0.1.1` here, or whatever version you're currently at. I'm guessing this is useful for backwards compatibility stuff, like when to make cops enabled by default or things like that. But using `rubocop` versioning here is completely meaningless, because it's your library adding this cop, not the core gem.

Reply:
Oh god, you're right!
Don't know what on earth was I thinking :man_facepalming: 

You're really a very nice reviewer, indeed! Thank you! :heart:

Reply:
(and I am equally a stupid committer :man_facepalming:)
Should be good now! :sweat_smile:

Reply:
Actually, you should include there the version where you plan to release this cop, not the current version :)

Reply:
D'oh :face_with_head_bandage: